src/screens/main/menu/contacts/ContactsTab.js

```diff
@@ -0,0 +1,205 @@
+import React, { Component } from 'react';
+import { View, Text, StyleSheet, ListView } from 'react-native';
+import { NavigationActions } from 'react-navigation';
+import { Icon, Button, FormLabel, FormInput, FormValidationMessage, List, ListItem } from 'react-native-elements';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import AddFirstContact from './add/AddFirstContact';
+import BackWithMenuNav from "../../../../components/customPageNavs/BackWithMenuNav"
+import ContactTabNavigator from '../../../../components/customPageNavs/ContactTabNavigator'
+import SelectedContact from './SelectedContact'
+
+
+/**
+ * Screen that displays all the contacts that have been added to
+ * the wallet
+ */
+class ContactsTab extends Component {
+
+  /**
```

Comment:
If the commented code is not going to be used in future then please clean it up.